### PR TITLE
[changelog] adding 6.5.0 changeset

### PR DESCRIPTION
**Pull request by truthbk**
### What does this PR do?

Updates the changelog

### Motivation

6.5.0 release

### CODE REVIEW

**Review by derekwbrown — DISMISSED** [review]

File: releasenotes/notes/prelude-release-6.5.0-1599e595834e5098.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+prelude:
+    |
```
Comment (arbll) [inline releasenotes/notes/prelude-release-6.5.0-1599e595834e5098.yaml]: Missing a `-` here

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,199 @@
...
+- Docker image: handle docker/kubernetes secret files with a helper script
+
+- The Node Agent can rely on the Datadog Cluster Agent to collect Node Labels.
+
+- Improved ECS fargate tagging:
+  - Honor the ``docker_labels_as_tags`` option to extract custom tags
+  - Make the ``cluster_name`` tag shorter
+  - Add the ``short_image`` and ``container_id`` tags
+  - Remove some noisy tags
+  - Fix a lifecycle issue that caused missing tags
```
Comment (olivielpeau) [inline CHANGELOG.rst]: format of this list is incorrect in `rst`, should be fixed

**Review by olivielpeau — DISMISSED** [review]
just a nit, feel free to merge once addressed 🙇

File: CHANGELOG.rst (hunk 2 above)
Comment (truthbk) [reply CHANGELOG.rst]: Nested list with `*` - tested here: http://www.siafoo.net/reST.xml

**Pull request merged**